packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -15,15 +15,18 @@ We need to provide users the means for changing the widths of individual columns
 
 ### Expected Behavior
 
-The behavior that has been prescribed for column sizing is as follows:
+#### Column Sizing
 
 -   Columns should be able to be configured to either maintain a fixed width, or grow proportionally with the table such as when the window resizes causing the table width to increase. Tables can consist of columns that are configured as a mixture of the two modes.
 -   If a user drags a divider between two columns to the right, then the column on the left will grow larger, and the column on the right will grow smaller by the same pixel amount. Sub-behaviors to this are:
     -   If a shrinking column has reached its minimum pixel size or is not resizable, then the next column in the direction of the sizing action will be affected up to the final column in a given direction.
     -   A sizing action to the left will ultimately stop having an effect when the left-most column reaches its minimum size.
-    -   A sizing action to the right that would ultimately result in the final right column reaching its minimum size (all columns still within current table width) would then begin to push columns out of the table width resulting in a horizontal scrollbar.
+    -   A sizing action to the right that would ultimately result in the final right column reaching its minimum size (all columns still within current table width) would _conditionally_ (see behavior descriptions below) begin to push columns out of the table viewport width resulting in a horizontal scrollbar.
 -   Columns can be configured to not allow a user to interactively size them
     -   The implicit behavior present based on the behaviors described above, is that in a sizing action that cascades to a column configured to not be resized is that the column won't be resized towards a minimum size, and the cascade will effectively "skip" this column.
+-   A user will be able to configure the table such that dragging a column will never result in a change in the table's viewport width. This does _not_ mean that a table configured in such a will _not_ have a scrollbar (e.g. a table could still have enough columns in its view to demand a horizontal scrollbar). This API will simply prevent the user from expanding the existing viewport width.
```

Comment:
Configuring this behavior will be done via an API, not a user-visible setting, correct? If so I'd suggest rewording to say "A client will be able to configure..." or "The table will expose an API to configure it such that..."